src/QueryBuilder.php

```diff
@@ -86,7 +86,7 @@ public function whereIsRoot()
      */
     public function whereAncestorOf($id, $andSelf = false, $boolean = 'and')
     {
-        $keyName = $this->model->getKeyName();
+        $keyName = $this->model->getTable() . '.' . $this->model->getKeyName();
```

Comment:
There is a problem here, as `$keyName` is also used below on line 103 in subquery and this needs to be in form of really just the key column name. If the table name is specified here, it won't work as the table is referenced by alias `_` in the subquery. 

`$keyName` should stay as is and the table name should be added only on line 117, where it is the only place where it is actually needed and prevents `Column '...' in where clause is ambiguous .` errors.